pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: dartdoc
-version: 8.0.11
+version: 8.0.12-dev
```

Comment:
When changing this version, 3 other spots need to change:

* CHANGELOG.md (you did)
* dartdoc_options.yaml
* lib/src/version.dart

Reply:
Should I use `-dev`? We use it in pana for intermediate states that precedes release, but keep the final version in changelog. What is the rule here?

Reply:
We stick `-dev` in each of those 4 locations generally. But I can release 8.0.12 right after this is merged so that pub.dev is fixed. So you can use 8.0.12 in each location.

Reply:
Please do publish 8.0.12 (not urgent though, just so that we don't forget)